cmd/start.go

```diff
@@ -32,12 +32,17 @@ func init() {
 		},
 	}
 
-	cmd.PersistentFlags().StringVar(&opts.ListenAddr, "listen", opts.ListenAddr, "Address to listen on for metrics")
-	cmd.PersistentFlags().StringVar(&opts.Kubeconfig, "kubeconfig", opts.Kubeconfig, "Kubeconfig file to access a remote cluster (testing only)")
+	cmd.PersistentFlags().StringVar(&opts.ListenAddr, "listen", opts.ListenAddr, "Address to listen on for metrics.")
+	cmd.PersistentFlags().StringVar(&opts.Kubeconfig, "kubeconfig", opts.Kubeconfig, "Kubeconfig file to access a cluster where the CVO is reconciling manifests.")
+	cmd.PersistentFlags().StringVar(&opts.KubeconfigManagementCluster, "kubeconfig-management-cluster", opts.KubeconfigManagementCluster, "Kubeconfig file to access a cluster that manages the CVO.")
 	cmd.PersistentFlags().StringVar(&opts.NodeName, "node-name", opts.NodeName, "kubernetes node name CVO is scheduled on.")
 	cmd.PersistentFlags().BoolVar(&opts.EnableAutoUpdate, "enable-auto-update", opts.EnableAutoUpdate, "Enables the autoupdate controller.")
 	cmd.PersistentFlags().StringVar(&opts.ReleaseImage, "release-image", opts.ReleaseImage, "The Openshift release image url.")
 	cmd.PersistentFlags().StringVar(&opts.ServingCertFile, "serving-cert-file", opts.ServingCertFile, "The X.509 certificate file for serving metrics over HTTPS.  You must set both --serving-cert-file and --serving-key-file unless you set --listen empty.")
 	cmd.PersistentFlags().StringVar(&opts.ServingKeyFile, "serving-key-file", opts.ServingKeyFile, "The X.509 key file for serving metrics over HTTPS.  You must set both --serving-cert-file and --serving-key-file unless you set --listen empty.")
+	cmd.PersistentFlags().StringVar(&opts.PromQLTarget.QueryCaBundleFile, "prometheus-query-ca-bundle-file", opts.PromQLTarget.QueryCaBundleFile, "The service CA bundle file containing one or more X.509 certificate files for validating certificates generated from the service CA for the respective PromQL target.")
+	cmd.PersistentFlags().StringVar(&opts.PromQLTarget.QueryNamespace, "prometheus-query-namespace", opts.PromQLTarget.QueryNamespace, "The name of the namespace where the the PromQL query service resides.")
```

Comment:
Maybe I'd mention that this namespace is supposed to exist on the cluster behind the `--kubeconfig-management-cluster`

Reply:
Definitely wouldn't hurt!

Reply:
> I will suggest s/remote PromQL query service resides/PromQL query service resides/

Done :+1: Added in the new commit.